default/stringtables/en.txt

```diff
@@ -7186,7 +7186,7 @@ SP_SLY_DESC
 '''Description:
 The Sly live in gas giants. The gas giants are so foreign to most species in the universe that, for most of their history, they have been overlooked.
 Not being able to build devices, they couldn't leave their homeworld. Recently, they were able to obtain necessary equipment and technology for spacefaring from traders in exchange for their knowledge of history.
-Keeping a strict diet a Sly crew on gas giants actually can help in refinining fuel for their ship.
+On the right diet Sly excrement refined fuel. When staying on a gas giant, a Sly crew will create some fuel for the ship.
```

Comment:
, after diet
excrement -> excrete
remove "some"
When in a system with a gas giant, a Sly-crewed ship will regenerate fuel.